chainntnfs/txnotifier.go

```diff
@@ -46,10 +46,22 @@ var (
 	// with the TxNotifier but it been shut down.
 	ErrTxNotifierExiting = errors.New("TxNotifier is exiting")
 
-	// ErrTxMaxConfs signals that the user requested a number of
-	// confirmations beyond the reorg safety limit.
-	ErrTxMaxConfs = fmt.Errorf("too many confirmations requested, max is %d",
-		MaxNumConfs)
+	// ErrNoScript is an error returned when a confirmation/spend
+	// registration is attempted without providing an accompanying output
+	// script.
+	ErrNoScript = errors.New("an output script must be provided")
+
+	// ErrNoHeightHint is an error returned when a confirmation/spend
+	// registration is attempted without providing an accompanying height
+	// hint.
+	ErrNoHeightHint = errors.New("a height hint greater than 0 must be " +
+		"provided")
+
+	// ErrNumConfsOutOfRange is an error returned when a confirmation/spend
+	// registration is attempted and the number of confirmations provided is
+	// out of range.
+	ErrNumConfsOutOfRange = fmt.Errorf("number of confirmations must be "+
+		"between %d and %d", 0, MaxNumConfs)
```

Comment:
Between 0 an max means `>0 && <=max`. If 'between' is inclusive, it should be between 1 and max?

Reply:
Yes, fixed.